src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -128,189 +64,142 @@ const AdvancedSettings: React.FC<AdvancedSettingsProps> = ({
-        {errors.radioStations && (
-          <span className="input-error">{errors.radioStations.message}</span>
-        )}
-      </Form.Group>
-    </ToggleElement>
-  );
-
-  const renderShowNametags = () => (
-    <S.ItemWrapper>
-      <S.ItemHeader>
-        <S.TitleWrapper>
-          <S.ItemTitle>Show Nametags</S.ItemTitle>
-        </S.TitleWrapper>
-
-        <S.ItemSubtitle>Display user names on their avatars</S.ItemSubtitle>
-      </S.ItemHeader>
-
-      <S.ItemBody>
-        <Form.Control as="select" custom name="showNametags" ref={register}>
-          <option value="none">None</option>
-          {/* TODO: Implement Inline state */}
-          {/* <option value="inline">Inline</option> */}
-          <option value="hover">Inline and hover</option>
-        </Form.Control>
-      </S.ItemBody>
-    </S.ItemWrapper>
-  );
-
-  const renderRoomVisibility = () => (
-    <S.ItemWrapper>
-      <S.ItemHeader>
-        <S.TitleWrapper>
-          <S.ItemTitle>Room appearance</S.ItemTitle>
-        </S.TitleWrapper>
-
-        <S.ItemSubtitle>
-          Choose how you&apos;d like your rooms to appear on the map
-        </S.ItemSubtitle>
-      </S.ItemHeader>
-
-      <S.ItemBody>
-        <Form.Control as="select" custom name="roomVisibility" ref={register}>
-          <option value="hover">Hover</option>
-          <option value="count">Count</option>
-          <option value="count/name">Count and names</option>
-        </Form.Control>
-      </S.ItemBody>
-    </S.ItemWrapper>
-  );
-
-  const renderAttendeesTitleInput = () => (
-    <S.ItemWrapper>
-      <S.ItemHeader>
-        <S.TitleWrapper>
-          <S.ItemTitle>Title of your venues attendees</S.ItemTitle>
-        </S.TitleWrapper>
-        <S.ItemSubtitle>
-          For example: guests, attendees, partygoers.
-        </S.ItemSubtitle>
-      </S.ItemHeader>
-
-      <S.ItemBody>
-        <Form.Control
-          name="attendeesTitle"
-          placeholder="Attendees title"
-          ref={register}
-          custom
-          type="text"
-        />
-        {errors.attendeesTitle && (
-          <span className="input-error">{errors.attendeesTitle.message}</span>
-        )}
-      </S.ItemBody>
-    </S.ItemWrapper>
-  );
-
-  const renderChatTitleInput = () => (
-    <S.ItemWrapper>
-      <S.ItemHeader>
-        <S.TitleWrapper>
-          <S.ItemTitle>Your venue chat label</S.ItemTitle>
-        </S.TitleWrapper>
-        <S.ItemSubtitle>For example: Party, Event, Meeting</S.ItemSubtitle>
-      </S.ItemHeader>
-
-      <S.ItemBody>
-        <Form.Control
-          name="chatTitle"
-          placeholder="Event label"
-          ref={register}
-          custom
-          type="text"
-        />
-        {errors.chatTitle && (
-          <span className="input-error">{errors.chatTitle.message}</span>
-        )}
-      </S.ItemBody>
-    </S.ItemWrapper>
-  );
-
   return (
-    <div>
+    <div className="AdvancedSettings">
       <h1>Advanced settings</h1>
```

Comment:
Can use components like

- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminPanel
- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminSidebar
- https://github.com/sparkletown/sparkle/tree/staging/src/components/organisms/AdminVenueView/components/AdminSidebarTitle

to unify admin UI/UX. Especially useful to not worry about proper semantic use of `h1`, `h2` etc.